### PR TITLE
programs page

### PR DESCRIPTION
**Pull request by kalidiagne**
- [x] Programs page
- [x] Program sections in homage
- [x] Mobile version
- [x] Sidebar 
- [x] Footer new design update
- [x] Content & Links
- [x] add open graph for programs page

### CODE REVIEW

**Comment by ChialiT** [comment]
on the dropdown menu in the hero section, they all open the same form? is that the final or?

**Comment by kalidiagne** [comment]
> on the dropdown menu in the hero section, they all open the same form? is that the final or?

Waiting for the final copy and links

File: data/programs/accelerationProgramFaq.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  },
+  {
+    question: "How does the grant funding work?",
+    answer: (
+      <>
+        <span>
+          While the core funding this program comes through PSE via the Ethereum
+          Ecosystem Support Program (ESP), some bounties are supported by
+          specific teams.
+        </span>
```
Comment (heeckhau) [inline data/programs/accelerationProgramFaq.tsx:22]: What are these spans supposed to do?
It does not look as expected here:
image

**Review by heeckhau — APPROVED** [review]
Looks good to me.

One extra remark: The fonts on the contribution Program cards look really tiny on my screen
image

File: data/programs/accelerationProgramFaq.tsx (hunk 1 above)
Comment (kalidiagne) [reply data/programs/accelerationProgramFaq.tsx:22]: Thank you for your feedback!
I have fixed the spacing, and the different `span` will correctly create a gap between the paragraphs

**Comment by kalidiagne** [comment]
PR update with final version for content

**Review by AtHeartEngineer — APPROVED** [review]

**Pull request merged**